packages/env/README.md

```diff
@@ -203,6 +203,14 @@ wp-env start
 wp-env start --xdebug=profile,trace,debug
 ```
 
+When you're running `wp-env` using `npm run`, like when working in the Gutenberg repo or when having `wp-env` as a local project dependency, don't forget to add an extra dash before the `--xdebug` command:
```

Comment:
```suggestion
When you're running `wp-env` using `npm run`, like when working in the Gutenberg repo or when having `wp-env` as a local project dependency, don't forget to add an extra double dash before the `--xdebug` command:
```

Reply:
Sure why not 🙂 Pushed a commit where I use "double dash", also at one pre-existing location.